tests/simulation/translator/ref/Flow360_CHT_three_cylinders.json

```diff
@@ -104,49 +104,19 @@
                     "lowMachPreconditionerSensor"
                 ]
             },
-            "solid-1/isothermal-1": {
+            "solid-1/Interface_fluid": {
                 "outputFields": [
-                    "Cp",
-                    "primitiveVars",
-                    "T",
-                    "heatFlux",
-                    "lowMachPreconditionerSensor"
+                    "T"
                 ]
             },
-            "solid-2/adiabatic-2": {
+            "solid-2/Interface_fluid": {
                 "outputFields": [
-                    "Cp",
-                    "primitiveVars",
-                    "T",
-                    "heatFlux",
-                    "lowMachPreconditionerSensor"
+                    "T"
                 ]
             },
-            "solid-2/isothermal-2": {
+            "solid-3/Interface_fluid": {
```

Comment:
Why the reference surface name  changed?

Reply:
Just realized these have been the wrong surfaces specified for output. For the CHT case we are only interested in the fluid-solid interfaces.